solidus_sale_prices.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
     s.metadata["source_code_uri"] = s.homepage if s.homepage
   end
 
-  s.required_ruby_version = '~> 2.4'
+  s.required_ruby_version = '~> 2.5'
```

Could we change this to allow Ruby 3 to be used as well? 
```suggestion
  s.required_ruby_version = Gem::Requirement.new('>= 2.5')
```